fluidreview/api.py

```diff
@@ -0,0 +1,138 @@
+"""
+FluidReview API backend
+"""
+from urllib.parse import urljoin
+from datetime import timedelta
+from django.conf import settings
+from requests_oauthlib import OAuth2Session
+from fluidreview.models import OAuthToken
+from fluidreview.utils import utc_now
+
+
+class FluidReviewAPI:
+    """
+    Class for making authorized requests to the FluidReview API via OAuth2
+    """
+
+    def __init__(self):
+        """
+        Prepare to make OAuth2 requests to the API
+        """
+        self.session = self.initialize_session()
+
+    def initialize_session(self):
+        """
+        Initialize an OAuth2 session with an auto-refreshing token.
+
+        Returns:
+            (OAuth2Session): an OAuth2 session
```

Comment:
Can you remove the parentheses here and below for the `Returns` section here and below?

Reply:
OK